### PR TITLE
Add user fetching and display

### PR DESCRIPTION
**Pull request by Bjornkjohnson**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,113 +1,31 @@
...
-              -&gt;
-            </span>
-          </h2>
-          <p className="m-0 max-w-[30ch] text-sm opacity-50">
-            Learn about Next.js in an interactive course with&nbsp;quizzes!
-          </p>
-        </a>
+export default async function Home() {
+  const users = await getUsers();
+  console.log(users);
```
Comment (AKnepshield) [inline src/app/page.tsx]: Could you please remove this console log

**Review by AKnepshield — CHANGES_REQUESTED** [review]
One minor update.  Otherwise looks good.

File: src/app/page.tsx (hunk 1 above)
Comment (Bjornkjohnson) [reply src/app/page.tsx]: Good catch.

**Review by AKnepshield — APPROVED** [review]
Looks good

**Pull request merged**